### ISSUE TITLE
Nitrogen does not work with nemo (cinnamon)

### ISSUE THREAD

**Issue by xhpohanka** [l3ib/nitrogen#91]
When nemo is running nitrogen falls with following error
```
(nitrogen:3402): GLib-GIO-ERROR **: Settings schema 'org.nemo.desktop' does not contain a key named 'draw-background'
```
Also there is no dropdown menu for multiple monitors.
It was tested on Arch linux with official nitrogen package

**Comment by dglava** [comment]
Do you use Nemo on its own or with Cinnamon?
If you're just using Nemo, you probably have to disable the desktop icons feature. I use standalone Nemo with Nitrogen and I have no problems setting the wallpaper.

**Comment by xhpohanka** [comment]
Yes, I use it with Cinnamon. Disabling desktop icons does not help and moreover I don't want to do that, I'm used to have icons on my desktop :)

**Comment by daf** [comment]
@xhpohanka I honestly thought I had removed this piece of code, but it appears it's still in there.  I'll fix it for 1.6.1.

**Comment by xhpohanka** [comment]
Nitrogen does not fail with an error now, but unfortunately there is still no dropdown menu for selecting monitors and no wallpaper is drawn...

**Comment by daf** [comment]
Ok, the background not being drawn is concerning - sounds like it's not setting something correctly for Nemo to pick up.

There should not be a monitor selection dropdown for any of the GNOME-like setting modes - at the time they didn't offer separate monitor wallpaper selection mechanisms and I haven't written anything to simulate it yet, so this is expected.  Can the native Nemo wallpaper config tools do separate monitor configs?

**Comment by xhpohanka** [comment]
Maybe now I understand it better.
I tried nitrogen because Cinnamon (or Nemo?) does not have support to set wallpaper per monitor. So I disabled `org.cinnamon.settings-daemon.plugins.background`

Now I see following behavior.

- Nemo killed (it means no icons on desktop) - nitrogen works, dropdown for monitors available
- Nemo running - no dropdown menu, nitrogen cannot draw wallpaper

With `org.cinnamon.settings-daemon.plugins.background` enabled nitrogen can draw wallpaper but only for both monitors simultaneously.

**Comment by daf** [comment]
Ok, thanks for the investigation. There's two things at play here.

The first is a bug with Nemo - it seems that if Nemo is running but does not have `org.cinnamon.settings-daemon.plugins.background` enabled, Nitrogen incorrectly detects how it should operate and tries to set Nemo gsettings which are being ignored.  I'll look into this and track that in this issue still.

The second is a now-planned feature: with `org.cinnamon.settings-daemon.plugins.background` enabled, we should simulate the multi-monitor ability even without support from Nemo: this will be a new feature, see #94.

Do I read your comment correctly, @xhpohanka?

**Comment by xhpohanka** [comment]
> Do I read your comment correctly, @xhpohanka?

Yes, you do...

**Comment by dgouramanis** [comment]
I am seeing the same symptoms, but different circumstances. I am also running Nemo + Cinnamon. No problems observed with Nemo. I used Nitrogen with no problems for over a year, running the machine as user 'root'. The drop down was enabled, and I was able to select different images on each monitor. 

When I made a new user (not root) the dropdown is disabled. Nitrogen failed to start. I got  following error. 

> (nitrogen:729): GLib-GIO-ERROR **: Settings schema 'org.nemo.desktop' does not contain a key named 'draw-background'
> Trace/breakpoint trap (core dumped)

Then I uninstalled Nitrogen, and reinstalled. Now nitrogen starts with the following warning:

> WARNING: More than one Desktop window found:
> **      0x 1a0000b   5        Nemo
>         0x 1a00003   5        Nemo

There is no drop-down menu for screen selection. It does work to display a single image on both screens.

**Comment by ShinigamiH4ck3r** [comment]
Dont mean to hijack anything but im getting the same error but on solus with the budgie desktop

> `(nitrogen:2232): GLib-GIO-ERROR **: Settings schema 'org.gnome.desktop.background' does not contain a key named 'draw-background' Trace/breakpoint trap`

**Comment by JSkier21** [comment]
Also have this error with xfce and thunar.

`(nitrogen:9511): GLib-GIO-ERROR **: Settings schema 'org.gnome.desktop.background' does not contain a key named 'draw-background'
Trace/breakpoint trap (core dumped)
`

**Comment by angiest** [comment]
> Dont mean to hijack anything but im getting the same error but on solus with the budgie desktop

Same on Artix Linux using Budgie.
